public/app/plugins/datasource/cloud-monitoring/datasource.ts

```diff
@@ -2,26 +2,27 @@ import _ from 'lodash';
 
 import {
   DataQueryRequest,
-  DataQueryResponseData,
-  DataSourceApi,
   DataSourceInstanceSettings,
   ScopedVars,
   SelectableValue,
-  toDataFrame,
+  DataQueryResponse,
 } from '@grafana/data';
 import { getTemplateSrv, TemplateSrv } from 'app/features/templating/template_srv';
 import { getTimeSrv, TimeSrv } from 'app/features/dashboard/services/TimeSrv';
 
 import { CloudMonitoringOptions, CloudMonitoringQuery, Filter, MetricDescriptor, QueryType } from './types';
-import { cloudMonitoringUnitMappings } from './constants';
-import API, { PostResponse } from './api';
+import API from './api';
```

Comment:
`PostResponse` was introduced [here](https://github.com/grafana/grafana/pull/28847) so it would good if @hugohaggmark could acknowledge that request cancellation is not affected by this PR's modifications.

Reply:
That change is Ok, and the cancellation seems to work as expected.